aws/import_aws_route53_zone_test.go

```diff
@@ -1,21 +1,26 @@
 package aws
 
 import (
+	"fmt"
 	"testing"
 
+	"github.com/hashicorp/terraform/helper/acctest"
 	"github.com/hashicorp/terraform/helper/resource"
 )
 
 func TestAccAWSRoute53Zone_importBasic(t *testing.T) {
 	resourceName := "aws_route53_zone.main"
 
+	rString := acctest.RandString(8)
+	zoneName := fmt.Sprintf("%s.terraformtest.com", rString)
```

Comment:
Unrelated note: Maybe we should consider making `terraformtest.com` a constant in the future and/or provide a helper function that returns a random "valid" domain name.

Reply:
🤔  Maybe we could...